spearhead/assets/sass/block-extends/_navigation.scss

```diff
@@ -0,0 +1,13 @@
+.wp-block-navigation {
+	.wp-block-navigation__container {
+		.wp-block-navigation-link {
+			margin: 0 calc( 0.5 * var(--global--spacing-unit) );
```

Comment:
I think this is a bit too much compared to what's on the original site (it goes into two lines)